Cue/app/actions/types.js

```diff
@@ -14,6 +14,10 @@ export type Action =
   | { type: 'DECK_EDITED', change: {} }
   | { type: 'SHARE_CODE_GENERATED', uuid: string, code: string }
   | { type: 'DECK_CONFLICT_RESOLVED', updatedDeck: Deck}
+	| { type: 'SEARCHED_DECKS', searchString: string, searchResults: Array<Deck>}
+	| { type: 'DECK_ALREADY_IN_LIBRARY'}
+	| { type: 'DECK_ADDED_TO_LIBRARY', addedDeck: Deck}
+	| { type: 'CLEAR_INACCESSIBLE_DECKS'}
```

Comment:
The indent in the file is really wonky, can you either update your new lines to match the wonky spacing or re-indent the file properly so we don't run into this again?